_style/declarations.md

```diff
@@ -160,7 +160,7 @@ or is of a non-functional nature (some mutable state, local or
 otherwise), the body must be enclosed in braces:
 
     def sum(ls: List[String]): Int = {
```

Comment:
The word "must" is perhaps too strong here without mentioning that it is a "must" in Scala 2 only... If there is no plan to make a specific style guide for Scala 3, perhaps it is best to include both Scala 2 and Scala 3 variations in style in the same guide; after all Scala 2 code will be around for almost eternity. I would thus suggest to include something about braces being optional in Scala 3 here.

Reply:
I'm considering Scala 3 out of scope for this PR, but I do hope somebody will tackle it in a future PR.